Added installation instructions and clarified usage (#19)
dims/maintainers: README.md

and print the github id, pr comment count and devstats contribution count as well.

## Installation

> Run `make` to install Maintainers. Currently `make` will run the `hack/install-maintainers.sh` script.

Reviewer: ```suggestion
Run `make` to install `maintainers`. Currently `make` will run the `hack/install-maintainers.sh` script.
```
nit

Author: Fixed!

## Usage

> Maintainers looks for OWNERS files in the repository where you run `maintainers`. In the

Reviewer: There are commands like `check-url` which don't need to interact with OWNERS files. Maybe we drop the Usage section or we could word it differently to be generic enough?

Author: I've made this more generic and not specific to OWNERS files. How does this new version look?

and print the github id, pr comment count and devstats contribution count as well.

## Installation

Run `make` to install `maintainers`. Currently `make` will run the `hack/install-maintainers.sh` script.

## Usage

Unless there is a flag to specify the context (e.g., --repository-github, --yaml-file,

> --kubernetes-directory), Maintainers uses the directory where you run `maintainers` to provide the context

Reviewer: ```suggestion
--kubernetes-directory), the tool uses the directory in which it is run to provide context
```

Author: I made this small change.
